### DISCUSSION TITLE
Version 1.25.1

### DISCUSSION THREAD

**Discussion by jangxx** [jangxx/netflix-1080p#33] [General]
- Fixed the audio and subtitle menu not showing all available options (Thanks @mprangenberg)
- Removed unused MSL client
- Minor code improvements

<hr /><em>This discussion was created from the release <a href='https://github.com/jangxx/netflix-1080p/releases/tag/v1.25.1'>Version 1.25.1</a>.</em>

**Comment by georgysaumen** [comment]
I cannot add to Chrome extension, they said this extension not available on Chrome Web Store

**Reply by jangxx** [reply]
https://github.com/jangxx/netflix-1080p/issues/32

**Comment by chrcoluk** [comment]
Hi I can no longer get the bitrate menu or the debug text to appear so I dont know if this is still working.

However toggling the 5.1 audio option does work for audio.

Is there a reason why I just get the right click browser menu now when right clicking the stream?

**Reply by jangxx** [reply]
I'm not sure what exactly you're talking about, did the right-click bring up some other menu before? I'm not aware of it ever doing that.

I also just tested opening the debug and info views with their respective shortcuts and both of them worked, so I can't reproduce your issue unfortunately. Are you sure the extension is installed correctly? Do you see the text `Loaded the patched cadmium player` in the JavaScript console?
